server/src/main/java/com/objectcomputing/checkins/services/team/Team.java

```diff
@@ -0,0 +1,64 @@
+package com.objectcomputing.checkins.services.team;
+
+import java.sql.Date;
```

Comment:
It looks like this import is unused. Can we remove this?

Reply:
Done